config/settings/production.yml

```diff
@@ -43,3 +43,26 @@
 # Social media connection information.
      facebook_app_id: <%= ENV['FACEBOOK_APP_ID'] %>
      share_progress_api_key: <%= ENV['SHARE_PROGRESS_API_KEY'] %>
+
+
+# Braintree
+     braintree:
+       merchant_id: <%= ENV['BRAINTREE_MERCHANT_ID'] %>
+       public_key: <%= ENV['BRAINTREE_PUBLIC_KEY'] %>
+       private_key: <%= ENV['BRAINTREE_PRIVATE_KEY'] %>
+
+       merchants:
+         EUR: <%= ENV['BRAINTREE_MERCHANT_ID_EUR'] %>
```

Comment:
I feel like we're going to end up with an infinite number of ENV vars and would like to consider alternative ways to managing this (especially the subscription plan ids since they can be assigned reasonable IDs.